packages/flutter/test/material/expansion_tile_test.dart

```diff
@@ -256,4 +256,99 @@ void main() {
     expect(listTileRect.top, tallerWidget.top - remainingHeight / 2 - 12);
     expect(listTileRect.bottom, tallerWidget.bottom + remainingHeight / 2 + 10);
   });
+
+  testWidgets('ExpansionTile alignment test', (WidgetTester tester) async {
```

Comment:
Can we also add a test for each of the default ExpansionTile alignment and crossAxisAlignment properties as well?